### PR TITLE
Refactor docs

### PR DESCRIPTION
**Pull request by adborden**
- Clarify between Terraform/Terragrunt versions
- Move MFA instructions to the top level
- Clarify instructions for Terraform vs Ansible
- Create environment-specific env.samples
- Add environment specific READMEs

### CODE REVIEW

**Comment by adborden** [comment]
Maybe some of the jumpbox instructions should be moved to datagov-deploy, since it's really setup for that project :thinking:

**Comment by adborden** [comment]
I think this is ready for review.

File: docs/datagov-deploy-environments.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+# ~/.ssh/config
+
+Host *.datagov.us
+    ForwardAgent yes
+```
+
+
+### Secrets
+
+Copy the `env.sample` to `.env`. Then you can populated these secrets from the
```
Comment (mogul) [inline docs/datagov-deploy-environments.md]: s/populated/populate/

**Comment by adborden** [comment]
Ready for another look.

**Pull request merged**